<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>G-Pad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="GGraphicsView" name="graphicsView">
      <property name="renderHints">
       <set>QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
      <property name="cacheMode">
       <set>QGraphicsView::CacheNone</set>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::AnchorViewCenter</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>563</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuG_pad">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentFiles"/>
    <addaction name="actionReload"/>
    <addaction name="actionClose"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFilter"/>
   </widget>
   <addaction name="menuG_pad"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="codeWidget">
   <property name="font">
    <font>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string> G-Code</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="GTableView" name="codeView">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="movesWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string> Moves</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="GTableView" name="movesView">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>10</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="navigatorWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Navigator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="GNavigatorView" name="navigatorView">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="fileopen"/>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="exit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="g-pad.qrc">
     <normaloff>:/icons/visible.svg</normaloff>:/icons/visible.svg</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Filter visible</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset theme="reload"/>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GTableView</class>
   <extends>QTableView</extends>
   <header location="global">gtableview.h</header>
  </customwidget>
  <customwidget>
   <class>GGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">ggraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>GNavigatorView</class>
   <extends>QTreeView</extends>
   <header location="global">gnavigatorview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="g-pad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
